core/src/test/java/io/questdb/test/griffin/OrderByTimeoutTest.java

```diff
@@ -97,18 +97,19 @@ public void tearDown() throws Exception {
 
     @Test
     public void testTimeoutLimitedSizeSortedLightRecordCursor() throws Exception {
+        setProperty(PropertyKey.CAIRO_SQL_PARALLEL_TOP_K_ENABLED, "false");
```

Comment:
this test is flapping for me

Reply:
Fixed in 3ed278b0cc8039770e5418bc7b398c6e71ba6cba